### PR TITLE
PMM-5558 added alertmanager

### PR DESCRIPTION
**Pull request by Sevlin**
Added alertmanager to ansible playbook

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/pmm-update?pullRequest=84) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona/pmm-update?pullRequest=84) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona/pmm-update?pullRequest=84) it.</sub>

**Comment by AlekSi** [comment]
Please do not merge yet.

File: ansible/playbook/tasks/update.yml [hunk 1]
```diff
@@ -116,6 +117,11 @@
     - name: Check reread results
       debug: var=reread_result.stdout_lines
 
+    - name: Create working directory for Alertmanager
+      file: path=/srv/alertmanager state=directory owner=pmm group=pmm
```
Comment (AlekSi) [inline ansible/playbook/tasks/update.yml:121]: Mentioned in the other PR that it actually should be `/srv/alertmanager/data`

Comment (AlekSi) [reply ansible/playbook/tasks/update.yml]: @EvgeniyPatlan @hors @Sevlin Do you remember why we decided to do it there, not in RPM spec?

**Pull request merged**